packages/orchestrator/internal/sandbox/fc/client.go

```diff
@@ -31,24 +33,33 @@ func newApiClient(socketPath string) *apiClient {
 
 func (c *apiClient) loadSnapshot(
 	ctx context.Context,
+	tracer trace.Tracer,
```

Comment:
There should no longer be a need to pass around a tracer, we can instantiate it once in each package, and use that:

```
var tracer = otel.Tracer("orchestrator.internal.sandbox.fc")
```